groundstation/backend_api/user.py

```diff
@@ -100,6 +160,7 @@ def __init__(self):
         super(UserList, self).__init__()
 
     @create_context
+    @login_required
     def get(self, local_args=None):
         """Endpoint for getting a list of users
```

Comment:
```suggestion
        """Endpoint for getting a list of users an admin user has created.
```
Update docstring to describe its new function, since it doesn't fetch all users anymore (not like I'd imagine we'd need an endpoint for that, but thats a future problem if we ever do find a reason of needing it).